jobs/director/templates/director.yml.erb.erb

```diff
@@ -189,6 +189,12 @@ if p('blobstore.provider') == "s3"
    params['blobstore']['options']['s3cli_path'] = "/var/vcap/packages/s3cli/bin/s3cli"
 end
 
+if p('blobstore.provider') == "gcs"
```

Comment:
Looks like I missed this. Is it fine to defer making this change until bosh-gcscli stabilizes the configuration file a bit more?